src/redisraft.h

```diff
@@ -700,6 +701,17 @@ typedef struct JoinLinkState {
     void (*complete_callback)(RaftReq *req);
 } JoinLinkState;
 
+typedef struct MultiState {
+    RaftRedisCommandArray cmds;
+    bool active;
+    bool error;
+} MultiState;
+
+typedef struct ClientState {
+    MultiState multiState;
+    bool asking;
+} ClientState;
```

Comment:
I feel like we'll use client struct a lot in future, I'd use a shorter name, just "struct Client" probably.

Reply:
I personally like ClientState, I think the little bit more descriptive is fine.

Reply:
ok